File: packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsPackage.java

package expo.modules.contacts;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.core.ExportedModule;
import expo.core.BasePackage;

public class ContactsPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new ContactsModule(context));
  }
}

File: packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsPackage.kt
package expo.modules.contacts

import android.content.Context

import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class ContactsPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(ContactsModule(context))
}
